types/utxo_test.go

```diff
@@ -0,0 +1,77 @@
+package types
+
+import (
+	"fmt"
+	"github.com/stretchr/testify/require"
+	"testing"
+
+	"github.com/FourthState/plasma-mvp-sidechain/utils"
+	"github.com/ethereum/go-ethereum/common"
+	ethcrypto "github.com/ethereum/go-ethereum/crypto"
+)
+
+// Basic tests checking methods for BaseUTXO
+func TestUTXO(t *testing.T) {
```

Comment:
Could we split this function into separate tests that independently test for UTXO functionality?